openlibrary/plugins/upstream/account.py

```diff
@@ -316,8 +319,10 @@ def POST(self):
                 return render['account/verify'](
                     username=f.username.value, email=f.email.value
                 )
-            except ValueError:
-                f.note = get_login_error('max_retries_exceeded')
+            except OLAuthenticationError as e:
+                f.note = get_login_error(e.__str__())
+                if sentry.enabled:
+                    sentry.capture_exception(e)
```

Comment:
We shouldn't need to call this manually; sentry under the hood should be capturing all these errors, even if they're `except`ed. You should be able to test this by hooking up your local environment to sentry.